dcsync.yaml

```diff
@@ -2,7 +2,7 @@
 title: T1003.006 - DCSYNC
 tactic:
   - TA0006
-technique: T1208
+technique: T1003.006 
```

Comment:
Looks like you added a trailing space here

Reply:
Yep fixed!